data/dbus/Makefile

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+BINDIR := /usr/bin
+DBUSSERVICESDIR := /usr/share/dbus-1/services
+
+SERVICES_GENERATED := $(patsubst %.service.in,%.service,$(wildcard *.service.in))
+SERVICES := ${SERVICES_GENERATED}
+
+%.service: %.service.in
+	cat $< | sed s:@bindir@:${BINDIR}:g | cat > $@
```

Comment:
@sed This is using the same as in data/systemd/Makefile as @mvo5 initially suggested :-)

Let me do the quoting.